.github/pr-labeler.yml

```diff
@@ -0,0 +1,2 @@
+# Apply label "v1" if target branch is "1.x"
+'v1': ['1.x']
```

Comment:
you shouldn't need these new files - we already have a labeler action, so I think appending this to https://github.com/PrefectHQ/prefect/blob/main/.github/labeler.yml will do the trick 👍